src/js/builtins/BundlerPlugin.ts

```diff
@@ -334,12 +335,12 @@ export function runOnResolvePlugins(this: BundlerPlugin, specifier, inputNamespa
   }
 }
 
-export function runOnLoadPlugins(this: BundlerPlugin, internalID, path, namespace, defaultLoaderId) {
+export function runOnLoadPlugins(this: BundlerPlugin, internalID, path, namespace, defaultLoaderId, isServerSide: boolean) {
```

Comment:
Maybe `isServerSide` could be an enum to clarify whether it means:
- RSC
- SSR
- !browser